ingesters/version/version.go

```diff
@@ -18,11 +18,11 @@ import (
 const (
 	MajorVersion = 5
 	MinorVersion = 8
-	PointVersion = 4
+	PointVersion = 5
 )
 
 var (
-	BuildDate time.Time = time.Date(2025, 7, 1, 23, 59, 59, 0, time.UTC)
+	BuildDate time.Time = time.Date(2025, 7, 24, 23, 59, 59, 0, time.UTC)
```

Comment:
are we really waiting until Thursday for a release? I don't like have the data in the future when we cut the release and I was thinking possibly as soon as Tuesday the 22nd...?